config/environments/development.rb

```diff
@@ -33,12 +33,8 @@
 
     config.middleware.use(Rack::Attack)
     config.action_mailer.default_url_options = { host: "localhost", port: 3000, protocol: "https" }
-    # Mailcatcher
-    config.action_mailer.delivery_method = :smtp
-    config.action_mailer.smtp_settings = {
-       port: Rails.configuration.pub_secrets[:smtp_server_port] || 1025,
-       address: Rails.configuration.pub_secrets[:smtp_server_address] || "127.0.0.1"
-    }
+    config.action_mailer.delivery_method = :letter_opener
```

Comment:
```suggestion
    config.action_mailer.delivery_method = :letter_opener_web
```

> ... if you really just want to browse mails using the web interface and don't care about opening emails automatically), you can set :letter_opener_web as your delivery method on your config/environments/development.rb:

https://github.com/fgrehm/letter_opener_web?tab=readme-ov-file#usage

If this will always run inside Docker or something, then Launchy will be unable to open the browser anyway. This prevents Launchy from trying to execute applications with questionable files inside /tmp.